### DISCUSSION TITLE
`ELF: not found` when running prisma cli

### DISCUSSION THREAD

**Discussion by judehunter** [prisma/prisma#2287] [Q&A] [answered]
I'm not sure if this is just my machine, but I'm getting:
```bash
$ npx prisma -v
Error: Command failed with exit code 2: .../prisma/node_modules/@prisma/cli/introspection-engine-debian-openssl-1.1.x --version  
.../prisma/node_modules/@prisma/cli/introspection-engine-debian-openssl-1.1.x: ELF: not found
.../prisma/node_modules/@prisma/cli/introspection-engine-debian-openssl-1.1.x: 2: Syntax error: ")" unexpected
```
When running any prisma command, like `prisma -v` and `prisma generate`.
My machine runs on FreeBSD; it's a hosting server.

Should I file a new bug issue? Or is it my fault it doesn't work?
Does prisma cli just not work on FreeBSD? Would generating on Windows and pushing to the repo work (would the server even start)?

**Comment by janpio** [comment, marked as answer]
That is definitely worth an issue so we can try to reproduce 👍

**Reply by judehunter** [reply]
Great, here it is: https://github.com/prisma/prisma/issues/2288

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
